homeassistant/components/garages_amsterdam/config_flow.py

```diff
@@ -0,0 +1,43 @@
+"""Config flow for Garages Amsterdam integration."""
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from . import get_coordinator
+from .const import DOMAIN  # pylint:disable=unused-import
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Garages Amsterdam."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    _options = None
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        errors = {}
+
+        if self._options is None:
+            self._options = {}
+            coordinator = await get_coordinator(self.hass)
+            for case in sorted(
+                coordinator.data.values(), key=lambda case: case.garage_name
+            ):
+                self._options[case.garage_name] = case.garage_name
```

Comment:
Do we need a dict if the keys and values are the same? It could just be a list then, I'm thinking.